src/main/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/config/RestControllerSecurityConfig.kt

```diff
@@ -53,7 +53,7 @@ class RestControllerSecurityConfig(private val config: AppConfiguration) : WebSe
 	}
 
 	@Bean
-	fun authenticationEntryPoint(): AuthenticationEntryPoint? {
+	fun authenticationEntryPoint(): AuthenticationEntryPoint {
 		val entryPoint = BasicAuthenticationEntryPoint()
```

Comment:
Liten endring som egentlig ikke er relatert til skedulering av re-start av poddene

Reply:
Riktig. Det blev visst en PR på q1 istedenfor branchen `self-destruct`. Sorry for ekstra stöy